modules/graphics/weston.nix

```diff
@@ -6,11 +6,21 @@
     driSupport = true;
   };
 
+  imports = [
+    ./weston.ini.nix
+    ./fonts.nix
+  ];
+
   environment.noXlibs = false;
   environment.systemPackages = with pkgs; [
     weston
     # Seatd is needed to manage log-in process for weston
     seatd
+
+    # Graphical applications
+    # Probably, we'll want to re/move it from here later
+    chromium
```

Comment:
Maybe we should move them quite soon, I just wanted to check few things from this PR by building it myself and testing, before giving approval. And now I'm probably going to have to wait at least ~4 hours building chromium from source

Reply:
I hope you used vedenemo cache, since chromium is already there=)